addons/Popochiu/Editor/Importers/Aseprite/docks/AnimationPlayerInspectorDock.gd

```diff
@@ -0,0 +1,330 @@
+tool
+extends PanelContainer
+
+const RESULT_CODE = preload("res://addons/Popochiu/Editor/Config/ResultCodes.gd")
+const LOCAL_OBJ_CONFIG = preload("res://addons/Popochiu/Editor/Config/LocalObjConfig.gd")
+# TODO: import a class without breaking coding standards... how?
+const AnimationTagRow = preload("res://addons/Popochiu/Editor/Importers/Aseprite/docks/AnimationTagRow.gd")
```

Comment:
@mapedorr I'm puzzled about this.

I need this const to "import" the `AnimationTagRow` class into the scope of this script, so I can explicit types (such as `var tag_row: AnimationTagRow` and invoke class methods `tag_row  = AnimationTagRow.new()' but since it's a const I should name it ANIMATION_TAG_ROW.

This makes Godot unable to find the class by class name though... I'm not sure what I am missing but what I need is to make Godot aware this class is among the others in the script and use it.
Maybe I'm just missing a `tool` statement (just thinking while I comment...)?

Reply:
In this case `AnimationTagRow` is the right way to name this constant. I'm not sure I understand when you say that Godot is unable to find the class by class name. Is it throwing and error? Or the autocomplete is not working?

Reply:
If I don't preload the class script that way, I can't use something like `var tag_row: AnimationTagRow` in my script.

If I preload the class script but name the constant UPPER_SNAKE_CASE_STYLE (like ANIMATION_PLAYER), I still can't `var tag_row: AnimationTagRow` in my script.

Godot has nothing like Namespaces or `use` statements, to my understanding so the only way to have ClassA available in ClassB.gd is 

```gdscript
# ...
ClassA = preload("res://path/to/ClassA.gd")
#...

var myClassAInstance: ClassA # <--- this won't work without the line above
myClassAInstance = ClassA.new() # <--- same
```

Reply:
Anyway if "class" constants are to be name after CamelCased classes, I think a specific constants section for them should be included and placed in the coding standards.

Reply:
I put a line of code in the coding standards for that, but you're right, it should have its own section to make it more clear:

```gdscript
# Use PascalCase when loading a class into a constant (or a variable)
const PopochiuClickable :=\
preload('res://addons/Popochiu/Engine/Objects/Clickable/PopochiuClickable.gd')
```